loaders/MiniMe/minime.ts

```diff
@@ -38,53 +41,96 @@ const ACRONYM_PARTS = `PC`;
 const FIELDS_ORDER = "id,nome,ordem,titulo&_where=ativo=true";
 const ACRONYM_ORDER = `TP`;
 
-const dollParts = {
-    pele: {
-        id: "",
-        img: "",
-    },
-    cabelo: {
-        id: "",
-        img: "",
-    },
-    face: {
-        id: "",
-        img: "",
-    },
-    roupa: {
-        id: "",
-        img: "",
-    },
-    acessórios: {
-        id: "",
-        img: "",
-    },
-    cheirinho: {
-        id: "",
-        img: ""
-    }
-}
 
-const loader = async () => {
-    
-    const customParts = await invoke.site.loaders.searchDocuments({
+const loader = async (
+    _props: DollParts,
```

Comment:
A prop não é utilizada, portanto, não faz sentido o tipo declarado.
```suggestion
    _props: unknown,
```